### ISSUE TITLE
Develop Budget

### ISSUE THREAD

**Issue by Allen616** [citizenlabsgr/community#28]

**Comment by BrandonKlotz** [comment]
I assume this is the expected 2017 Budget.

**Comment by Allen616** [comment]
Yes. Part of the formal 501c3 process is to provide a budget. Even if we go with the EZ form, it is a good thing for us to have.

**Comment by BrandonKlotz** [comment]
Is this resolved as a part of the response to the IRS? or do we need to develop this more?

**Comment by joel6653** [comment]
Pretty sure this is not temporary, but an ongoing thing. I suppose you could call it "developed" for the purpose of closing this task. But expect more work and status.

**Comment by Allen616** [comment]
I believe that we do need a working budget and the ability to track expenses. I will remove the 501c3 label from this task.

**Comment by joel6653** [comment]
I have the tracking software.

**Comment by BrandonKlotz** [comment]
@joel6653 what software do you use?

**Comment by joel6653** [comment]
I have used Banktivity (formerly iBank) for years for home. Also used it when I was treasurer at Wyoming Community Enrichment Commission. It will do what we need. It can import/export standard financial exchange formats.
